Source/WebCore/css/CSSCounterStyleDescriptors.cpp

```diff
@@ -229,4 +229,29 @@ CSSCounterStyleDescriptors CSSCounterStyleDescriptors::create(AtomString name, c
     return descriptors;
 }
 
+bool CSSCounterStyleDescriptors::areSymbolsValidForSystem() const
```

Comment:
Changes in this file look unrelated

Reply:
we are using this to invalidate a descriptor, I've clarified in the commit message, it is a small fix I've noticed when testing this patch